### PR TITLE
feat: runtime config

### PR DESCRIPTION
**Pull request by bason8800**

### CODE REVIEW

**Comment by Marcosaurios** [comment]
I was thinking about opening a new PR solving this issue, updating the plugin configuration in runtime. Thanks @bason8800 for the approach, I hope it gets merged!

**Comment by MrCode1997** [comment]
Hope, this will be merged soon!

**Comment by Marcosaurios** [comment]
To solve the problem with using multiple instances or versions of Google Recaptcha on the same page,I just discovered that if you need to change the plugin settings in runtime, you can do it updating plugin's object directly via `window.$nuxt.$recaptcha.version = 3` (in nuxt) or `this.$recaptcha` in vue. 

The point is if you update the plugin's `version` and `siteKey` you will need to force a render on the `<recaptcha />` plugin's component. I solved it adding the attribute `:key` (see [VUE docs :key attribute](https://v3.vuejs.org/guide/migration/key-attribute.html)) to the recaptcha component, and passing it the new updated value. Every time the `:key` property updates, the component will be rendered again. For example:
```
<recaptcha :key="this.$nuxt.$recaptcha.siteKey"
   @error="captchaV2Error"
   @success="captchaV2Success"
   @expired="captchaV2Expired"
 />
```

By this way, each time you update plugin's `version` and `siteKey` (in this order), the component will render again with the updated values from the plugin, so that you can use the same recaptcha component for testing V3 and V2 version.

I hope it helps as I have been struggling with this for almost a week.
@joelfloressottile @belgacea

**Comment by Atinux** [comment]
Thank you for this great PT @bason8800 

Do you mind adding a note into the README to show how to use the publicRuntimeConfig too?

**Comment by bason8800** [comment]
@Atinux done

**Comment by MrCode1997** [comment]
Hi @farnabaz , with the @bason8800 change, we can use publicRuntimeConfig, to change the env var for sitekey without rebuild all project, but just launching the "npm start" command, we can have the var updated.
Thx

File: lib/plugin.js [hunk 1]
```diff
@@ -159,6 +159,12 @@ class ReCaptcha {
 }
 
 export default function (_, inject) {
+  const { recaptcha = {} } = _.$config
```
Comment (Atinux) [inline lib/plugin.js:162]:
```suggestion
  const { recaptcha = {} } = _.$config || {}
```

For retro-compatibility

File: README.md [hunk 2]
```diff
@@ -59,6 +59,18 @@ using top level options
 }
 ```
 
+## Runtime config
+```js
```
Comment (Atinux) [inline README.md:63]:
```suggestion

```js
```

**Comment by Atinux** [comment]
Changes looks good, I added two small code suggestions

**Comment by farnabaz** [comment]
@bason8800 Do you mind to resolve conflicts? (I don't have write permission of your repo)

**Comment by farnabaz** [comment]
Thank you @bason8800

**Pull request merged**